File: app-process-container-example-go/containers/AppExampleContainer.go
package containers

import (
	"github.com/exproletariy/pip-services3-containers-examples/app-process-container-example-go/build"
	cproc "github.com/pip-services3-go/pip-services3-container-go/container"
	rpcbuild "github.com/pip-services3-go/pip-services3-rpc-go/build"
)

type AppExampleContainer struct {
	cproc.ProcessContainer
}

func NewAppExampleContainer() *AppExampleContainer {
	c := AppExampleContainer{}
	c.ProcessContainer = *cproc.NewProcessContainer("app-example", "Example of process container")
	c.AddFactory(build.NewAppExampleServiceFactory())
	c.AddFactory(rpcbuild.NewDefaultRpcFactory())
	return &c
}

File: app-process-container-example-nodex/src/container/AppExampleContainer.ts
import { ProcessContainer } from 'pip-services3-container-nodex';
import { DefaultRpcFactory } from 'pip-services3-rpc-nodex';
import { AppExampleServiceFactory } from "../build";

export class AppExampleContainer extends ProcessContainer {
    public constructor() {
        super('app-example', 'Example of simple container');
        this._factories.add(new AppExampleServiceFactory());
        this._factories.add(new DefaultRpcFactory());
    }
}